_data/footer.yml

```diff
@@ -1,26 +1,65 @@
-# Whether to show the "Return to top" link; can also be
-# configured as an object with 'text' and 'href' properties.
-top: true
+# This is the configuration for the site footer.
 
-# Which links to show at the top of the footer.
-# This can key into _data/navigation.yml or a list of link objects
-# with 'text' and 'href' properties.
+# Configuration for type of footer.
+# Footer types can be only be medium at the moment
+type: medium
+
+# Configuration for "Return to top" link.
+# Customize with 'text' and 'href' properties.
+# Comment out to the following section to hide.
+top:
+  text: Return to top
+  href: '#'
+
+# Configuration for "Edit this page" link in the footer.
+# Customize with 'text' property.
+# Uncomment the following line to show.
+# edit_page:
+#   text: Edit this page
+
+# Configuration for "Last updated" date and time.
+# Uncomment the following line to show.
+# last_updated: true
+
+# Configuration for which links show at the top of the footer.
+# this is a key into _data/navigation.yml
 links: footer
 
-# The optional heading for the footer.
+# Configuration for footer heading. (optional)
 heading: Name of agency
 
-# An array of agency logos to show, side by side, in the lower left
-# of the footer on large screens.
+# Configuration for agency logo(s) (shown side by side).
+# If the logo is external add external: true
 logos:
-  - src: /assets/img/us_seal.png
-    alt: Your agency name
+  - src: https://18f.gsa.gov/assets/img/logos/18f-logo.svg
```

Comment:
why not use an internal 18F logo and an external GSA logo to show both ways?

Reply:
We can add both logos by default - that's an easy update.

Reply:
Let's do that - to show both ways to link an image.